core/services/relay/evm/chain_writer.go

```diff
@@ -104,7 +105,6 @@ func (w *chainWriter) Close() error {
 		_, cancel := context.WithTimeout(context.Background(), time.Second)
 		defer cancel()
 
-		// TODO(nickcorin): Add shutdown steps here.
 		return nil
```

Comment:
Are just clearing the TODO but still planning to add shutdown steps here later? If you don't anticipate needing to add anything here, just wanted to suggest to clean out the context that isn't being used here also.